.github/workflows/main.yml

```diff
@@ -32,6 +32,13 @@ jobs:
         run: yarn build:swagger-gen
       - name: Build
         run: yarn build
+      - name: zip dist content
+        run: zip -q -r dist.zip . -i "*/dist/*" -x "*/node_modules/*" ".cache/*"
```

Comment:
Could be more specific with the regexp form `workspaces` field: `apps/*/dist`, `packages/*/*/dist` and `packages/@o3r/amaterasu/*/dist`